src/api/app/controllers/webui/project_controller.rb

```diff
@@ -654,7 +654,9 @@ def load_project_info
     @linking_projects = @project.linked_by_projects.pluck(:name)
 
     reqs = @project.open_requests
```

Comment:
This line returns all open requests, without any distinction between incoming and outgoing requests.

Incoming requests are requests for which the target project is the project (`@project` in this context).

Outgoing requests are requests for which the source project is the project (`@project` in this context).

So with this in mind, we need to filter them depending on those conditions.

Reply:
@dmarcoux maybe filtering them, creating two instance methods in ```Project```, one named ```incoming_requests``` and other named ```outgoing_requests```  ?

Reply:
let me see if I understood correctly. Do you want to rewrite the open_requests method in project.rb in models and divide it into incoming and outgoing?
incoming_requests should contain review and targets and outgoing should contain incidents and maintenance? Or what should be in each one?

Reply:
`open_requests` is used in other files, so I would instead create 2 methods, one for incoming requests and another for outgoing requests.